commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -35,6 +38,24 @@
 
   @Spec CommandSpec spec;
 
+  @Command(name = "export", description = "Export slashing db to json file")
```

Comment:
Should be "slashing protection db" to be accurate and all. Probably should also mention that this in the interchange format.

Reply:
its probably a signed artefacts database to be honest... the fact that we use it for slashing protection is implementation detail